os/arch/Kconfig.chip

```diff
@@ -443,6 +443,13 @@ config ARCH_STACKDUMP
 	---help---
 		Enable to do stack dumps after assertions
 
+config DUMP_CALLSTACK_WITH_SYMBOLS
```

Comment:
Is there dump functionality on this? Or getting symbol only?

Reply:
getting symbol only